--- a/third_party/farmhash/farmhash.BUILD
+++ b/third_party/farmhash/farmhash.BUILD
@@ -8,6 +8,13 @@
     },
 )

+config_setting(
+    name = "windows_arm64",
+    values = {
+        "cpu": "arm64_windows",
+    },
+)
+
 cc_library(
     name = "farmhash",
     srcs = ["src/farmhash.cc"],
@@ -17,6 +24,7 @@
     copts = select({
         ":windows": ["/DFARMHASH_OPTIONAL_BUILTIN_EXPECT"],
+        ":windows_arm64": ["/DFARMHASH_OPTIONAL_BUILTIN_EXPECT"],
         "//conditions:default": [],
     }),
     includes = ["src/."],
     visibility = ["//visibility:public"],
