--- a/third_party/py/python_configure.bzl
+++ b/third_party/py/python_configure.bzl
@@ -155,11 +155,8 @@
             python_bin,
             "-Wignore",
             "-c",
-            "import importlib; " +
-            "import importlib.util; " +
-            "print(importlib.import_module('distutils.sysconfig').get_python_inc() " +
-            "if importlib.util.find_spec('distutils.sysconfig') " +
-            "else importlib.import_module('sysconfig').get_path('include'))",
+            # distutils is not present in Python 3.12
+            "import sysconfig; print(sysconfig.get_path('include'))",
         ],
         error_msg = "Problem getting python include path.",
         error_details = ("Is the Python binary path set up right? " +
