--- a/tensorflow/BUILD
+++ b/tensorflow/BUILD
@@ -331,6 +331,19 @@
 )

 config_setting(
+    name = "windows_arm64",
+    flag_values = if_google(
+        {"//tools/cpp:cc_target_os": "windows"},
+        {},
+    ),
+    values = if_oss(
+        {"cpu": "arm64_windows"},
+        {},
+    ),
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
     name = "no_tensorflow_py_deps",
     define_values = {"no_tensorflow_py_deps": "true"},
     visibility = ["//visibility:public"],
